app/main.py

```diff
@@ -12,6 +12,8 @@
 from ytdl import DownloadQueueNotifier, DownloadQueue
 
 log = logging.getLogger('main')
+if __name__ == '__main__':

```

Comment:
Why has this moved here? There's no reason to move it...

Reply:
Because the call to Config comes before the logging level is set as such if you use debug level it wont show up. at least from my testing.

Reply:
@alexta69  I can move it back but the logging in the Config class will have to be changed to INFO or removed as it's won't show up otherwise.